test/templates/layout.njk

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ id }}</title>
     <link rel="stylesheet" href="static/layout.css">
+    <link rel="stylesheet" href="sap-theming/css_variables.css">
```

Comment:
Until we have PostCSS working in the playground, this will load the Fiori 3 CSS variables.